src/Ledger/Utxo.lagda

```diff
@@ -454,7 +454,7 @@ data _⊢_⇀⦇_,UTXO⦈_ where
     ∙ coin mint ≡ 0                          ∙ txsize ≤ maxTxSize pp
 
     ∙ ∀[ (_ , txout) ∈ txoutsʰ .proj₁ ]
-        inject (utxoEntrySize txout * minUTxOValue pp) ≤ᵗ getValueʰ txout
+        inject (utxoEntrySize txout * coinsPerUTxOByte pp) ≤ᵗ getValueʰ txout
```

Comment:
As I pointed out in the ticket this is also wrong. @williamdemeo, I am not sure if you plan to fix it in a separate PR, but the correct functionality is described in Babbage spec: https://github.com/IntersectMBO/cardano-ledger/blob/8d16d698dd499fd171845da16611c7c7026c1754/eras/babbage/formal-spec/utxo.tex#L194

On the up side, this is already closer to truth than how it was prior to PR :grin: 

For reference here is ledger implementation: https://github.com/IntersectMBO/cardano-ledger/blob/8d16d698dd499fd171845da16611c7c7026c1754/eras/babbage/impl/src/Cardano/Ledger/Babbage/TxOut.hs#L595-L618

Reply:
Yeah, I think that needs to be another PR, now is not the time to worry about serialization issues.

Reply:
It is not about serialization, but rather about the formula that is being used for computing the min value.
We can deal with size of TxOut in the same way we deal with hashes in conformance. We can just pass it when translating it to MAlonzo

Reply:
Yeah, that's what I mean. Implementing that is a bunch of work and there are already too many high-priority things.

Reply:
Yeah, seems like we are short staffed on the formal methods side.